common/components/controllers/LogInController.jsx

```diff
@@ -41,6 +41,13 @@ class LogInController extends React.Component<Props, State> {
     }
   }
 
+  //if the user is logged in, navigate back to home page
+  componentDidMount():void{
+    if (CurrentUser.isLoggedIn()) {
+      document.location.href = url.section(Section.Home);
+    }
```

Comment:
Let's make this its own function in [url.js](https://github.com/DemocracyLab/CivicTechExchange/blob/2a53c9645944133f6964cde357eed18d309b1f74/common/components/utils/url.js#L70)